### ISSUE TITLE
Godot crash with signal 4 when selecting a node in the inspector(?)

### ISSUE THREAD

**Issue by QbieShay** [godotengine/godot#24758]
**Godot version:**
11d7738

**OS/device including version:**
4.19.8-arch1-1-ARCH
NVIDIA Corporation GP106 [GeForce GTX 1060 6GB]
Driver Version: 415.22

**Issue description:**
When selecting a node in the inspector (inherited from an escn file, might be relevant) Godot crashed with signal 4.
It happened only once.

```
[1] /usr/lib/libc.so.6(+0x37e00) [0x7fd43bc60e00] (??:0)
[2] CowData<Ref<Image> >::get(int) const (/./core/cowdata.h:149)
[3] Vector<Ref<Image> >::operator[](int) const (/./core/vector.h:89)
[4] RasterizerStorageGLES3::texture_get_data(RID, int) const (/drivers/gles3/rasterizer_storage_gles3.cpp:1036)
[5] VisualServerRaster::texture_get_data(RID, int) const (/servers/visual/visual_server_raster.h:153)
[6] CommandQueueMT::CommandRet2<VisualServer, Ref<Image> (VisualServer::*)(RID, int) const, RID, int, Ref<Image> >::call() (/./core/command_queue_mt.h:307)
[7] CommandQueueMT::flush_one(bool) (/./core/command_queue_mt.h:426)
[8] CommandQueueMT::flush_all() (/./core/command_queue_mt.h:465)
[9] VisualServerWrapMT::sync() (/servers/visual/visual_server_wrap_mt.cpp:92)
[10] Main::iteration() (/main/main.cpp:1878)
[11] OS_X11::run() (/platform/x11/os_x11.cpp:2821)
[12] ./godot.x11.tools.64.llvm(main+0x151) [0x14ebe01] (/platform/x11/godot_x11.cpp:55)
[13] /usr/lib/libc.so.6(__libc_start_main+0xf3) [0x7fd43bc4d223] (??:0)
[14] ./godot.x11.tools.64.llvm(_start+0x2e) [0x14ebbee] (??:?)
```

**Steps to reproduce:**
Don't know, sorry.

**Comment by bojidar-bg** [comment]
CC @hpvb, because it crashed in CowData.

**Comment by akien-mga** [comment]
Duplicate of https://github.com/godotengine/godot/issues/24635#issuecomment-450342614, fixed by #24651.
